modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -153,6 +158,12 @@ func (ash *Handler) Provision(ctx caddy.Context) error {
 		return fmt.Errorf("certificate lifetime (%s) should be less than intermediate certificate lifetime (%s)", time.Duration(ash.Lifetime), time.Duration(ca.IntermediateLifetime))
 	}
 
+	repl, ok := ctx.Context.Value(caddy.ReplacerCtxKey).(*caddy.Replacer)
```

Comment:
Doesn't matter at all, but can simplify this

```suggestion
	repl, ok := ctx.Value(caddy.ReplacerCtxKey).(*caddy.Replacer)
```

We can simplify it in `caddyevents/app.go` as well, which I assume is where this was taken from.

Reply:
Oops, copy-paste mistake. Thanks for catching it.

I keep searching for this line every time I need the `Replacer`. We probably need to add a method for it on the `caddy.Context`.